data/lm/generate_lm.py

```diff
@@ -4,74 +4,97 @@
 import os
 import subprocess
 from collections import Counter
+from itertools import zip_longest
+import shutil
 
 import progressbar
 
 
 def convert_and_filter_topk(args):
     """ Convert to lowercase, count word occurrences and save top-k words to a file """
 
-    counter = Counter()
-    data_lower = os.path.join(args.output_dir, "lower.txt.gz")
+    # Loop over multiple input_txt arguments
+    data_lowers = []
+    vocabs = []
+    vocab_str_combined = ""
+    fill = args.top_k[-1] # with multiple input_txt arguments where there are fewer top_k
```

Comment:
I feel it's a hard to read design for users of the code, and likely to be a can of worm to deal with, leading to weird behaviors